src/spdx/clitools/pyspdxtools.py

```diff
@@ -26,7 +26,7 @@
 @click.command()
 @click.option("--infile", "-i", prompt="input file path", help="The file containing the document to be validated or converted.")
```

Comment:
Could you please also delete the prompt? I think we can do both of these changes within one PR that touches CLI.

Reply:
deleted